docs/tutorials/gromacs/spack-gromacs.yaml

```diff
@@ -107,22 +107,27 @@ deployment_groups:
     source: modules/scripts/startup-script
     settings:
       runners:
+      - type: shell
+        destination: remove_lustre_client.sh
+        content: |
+          #!/bin/bash
+          rm /etc/yum.repos.d/lustre-client.repo
```

Comment:
I know this is probably not the only place but we need to document why we are doing this.

I would recommend we add a section to `docs/slurm-troubleshooting.md` and explain the issue. Then leave a comment on each of these startup scripts that points to the documentation.

Reply:
I am not sure if we should add this. Looks like slurm-troubleshooting doc is if there is any issue with deploying the cluster, how can customer navigate it. 
This script removes lustre client because of known issue we found out in slurm image. Should we put all the details over there  ?

Reply:
My concern is that a customer will look at this line in the blueprint and have no idea why it is needed. Can we give the user some more information that this is a workaround to a known issue. I am not sure the best form for that to take but I would probably like some reference to the problem in the blueprint since that is where the confusion is likely to occur.

Reply:
Added a comment in the beginning. After we move to 6.3.1 I will remove this additional script from all BPs.